src/pages/module-deactivation/index.js

```diff
@@ -0,0 +1,181 @@
+// ** React Imports
+import { useEffect, useState, useContext } from 'react'
+
+// ** MUI Imports
+import { Box } from '@mui/material'
+
+// ** Third Party Imports
+import { useFormik } from 'formik'
+
+// ** Custom Imports
+import Table from 'src/components/Shared/Table'
+import WindowToolbar from 'src/components/Shared/WindowToolbar'
+import CustomTabPanel from 'src/components/Shared/CustomTabPanel'
+
+// ** API
+import { RequestsContext } from 'src/providers/RequestsContext'
+
+// ** Helpers
+import ErrorWindow from 'src/components/Shared/ErrorWindow'
+import { useResourceQuery } from 'src/hooks/resource'
+
+// ** Resources
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { AccessControlRepository } from 'src/repositories/AccessControlRepository'
+import { useWindowDimensions } from 'src/lib/useWindowDimensions'
+import { AuthContext } from 'src/providers/AuthContext'
+import { DataSets } from 'src/resources/DataSets'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import toast from 'react-hot-toast'
+
+const ModuleDeactivation = () => {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { height } = useWindowDimensions()
+
+  //states
+  const [errorMessage, setErrorMessage] = useState(null)
+  const { user } = useContext(AuthContext)
+
+  async function getGridData(options = {}) {
+    const languageId = user.languageId
+    const datasetId = DataSets.MODULE
+
+    const resModule = await getRequest({
+      extension: SystemRepository.KeyValueStore,
```

Comment:
please use the  getAllKvsByDataset function for getting data from SY.qryKVS if it's not a combobox:

ex:
 const fillTypeStore = () => {
    getAllKvsByDataset({
      _dataset: DataSets.RT_Product_Type,
      callback: setTypeStore
    })
  }